policyengine_canada/variables/household/person/childcare_expense.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class childcare_expense(Variable):
```

Comment:
Since we already have a household-level `childcare_costs` variable, I filed https://github.com/PolicyEngine/policyengine-canada/issues/307 to unify logic toward this.